llamafile/flags.cpp

```diff
@@ -137,6 +138,13 @@ void llamafile_get_flags(int argc, char **argv) {
             continue;
         }
 
+        if (!strcmp(flag, "-u") || !strcmp(flag, "--url-prefix")) {
```

Comment:
Please remove `-u`. If that flag is also supported in the old server then please remove it there too.

Reply:
Done